README.md

```diff
@@ -28,12 +28,29 @@ In alphabetical order:
 
 ## Installation
 
-Currently, the package is published on PyPI. To install the package, run the following command:
+### For Regular Users
+
+Currently, the package is published on PyPI. If you're looking to use VisuMorph without diving into the development, you can easily install it using pip:
 
 ```bash
 pip install visumorph
 ```
 
+### For Developers
+
+For those interested in contributing or exploring the development version, please follow these steps:
+
+1. Clone the repository:
+   ```bash
+   git clone https://github.com/UBC-MDS/VisuMorph.git
+   ```
+
+2. Navigate to the cloned directory and install the package in editable mode along with the development dependencies:
+   ```bash
+   pip install -e ".[dev]"
```

Comment:
Have you tested if this works? We have no setup.py.

Reply:
Yes, I checked and it is working. @Marcony1 can you please check this one on your device as well?

Reply:
I created a clean VM and confirm the line works - But please use `.` instead of `".[dev]"` as our package does not provide optional `dev` dependencies right now. It can also be noted that from the warning message during installation:

<img width="549" alt="image" src="https://github.com/UBC-MDS/VisuMorph/assets/17952490/a3afc6fb-7eb8-4a04-bb3d-2782a56c6e58">

Reply:
```suggestion
   pip install -e .
```